hal/src/dmac/channel/mod.rs

```diff
@@ -145,7 +145,7 @@ impl<Id: ChId, S: Status> Channel<Id, S> {
         // Software reset the channel for good measure
         self._reset_private();
 
-        #[cfg(any(feature = "samd11", feature = "samd21"))]
+        #[cfg(any(feature = "samd11", feature = "samd20", feature = "samd21"))]
```

Comment:
To better match the `#[cfg(feature = "min-samd51g")]` below, I would prefer `#[cfg(not(feature = "min-samd51g"))]` here.

Reply:
Same, updated